client/js/_author/components/assessments/navigation_bar_content.jsx

```diff
@@ -2,48 +2,66 @@ import React            from 'react';
 import _                from 'lodash';
 import { hashHistory }  from 'react-router';
 import Icon             from '../bank_navigation/bank_icon';
+import appHistory       from '../../history';
 
-export default function AssessmentsView(props) {
+export default class NavigationBarContent extends React.Component {
```

Comment:
Looking at this again, I kind of think it has enough logic that it makes sense to be a class. but if you feel strongly about it I can change it